src/test/java/org/alfresco/repo/rendition2/LocalTransformServiceRegistryIntegrationTest.java

```diff
@@ -92,30 +95,34 @@ protected boolean isEnabled()
     @Test
     public void testIsSupported()
     {
+        // Need to make sure we don't fall back to a transformer without limits.
+        List<SupportedTransform> transformers = localTransformServiceRegistry.findTransformers(MIMETYPE_IWORK_PAGES, targetMimetype, options, RENDITION_NAME);
+        assertEquals(1, transformers.size());
+
         // +ve
         // No props
-        Assert.assertTrue(transformServiceRegistry.isSupported(MIMETYPE_OPENXML_WORDPROCESSING, 1234, MIMETYPE_PDF, options, RENDITION_NAME));
+        Assert.assertTrue(transformServiceRegistry.isSupported(MIMETYPE_IWORK_PAGES, 1234, targetMimetype, options, RENDITION_NAME));
```

Comment:
The wildcard changes introduced another transformer from docx to pdf which did not have a size limit. As a result this test failed. Ended up having to create a 'dummy' test pipeline (only in the test context) that could be used, hence the need to change mimetypes.